pkg/storaged/vdo-details.jsx

```diff
@@ -199,6 +200,7 @@
                 Action: {
                     Title: _("Delete"),
                     Danger: _("Deleting erases all data on a VDO device."),
+                    disable_on_error: usage.Teardown,
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19579-20231108-113532-872713b2-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)